modules/imaging_browser/test/TestPlan.md

```diff
@@ -14,31 +14,31 @@
 8. Sidebar:  all links work, including the Download DICOM option. Ensure that projects can customize (add/remove) their list of instruments that can be linked to from the Configuration/Imaging Modules/Imaging Browser Links to Insruments
 9. 3d panel overlay etc - they work.  Add panel checkbox works. 3D only or 3D+Overlay loads files if at least one image exists and is selected
 10. "Visit Level Feedback" - pops up QC window (see section E below)
-11. Visit level QC controls (Pass/Fail, Pending, Visit Level Caveat) viewable to all, editable IFF permission imaging_browser_qc
-12. Save button appears IFF permission imaging_browser_qc
+11. Visit level QC controls (Pass/Fail, Pending, Visit Level Caveat) viewable to all, editable only if permission imaging_browser_qc
+12. Save button appears only if permission imaging_browser_qc
 13. Test Save button works 
 14. Verify Visit-level QC controls and comments can be deleted/unchecked/emptied and saved
 15. Test Breadcrumb link back to Imaging Browser
 
 ### C. Main panel:  per acquisition:
 
-16. Files can be downloaded (links clickable) only IFF has permission. Ensure that DICOM downloads are
+16. Files can be downloaded (links clickable) only if has permission. Ensure that DICOM downloads are
```

Comment:
```suggestion
16. Files can be downloaded (links clickable) only if user has permission. Ensure that DICOM downloads are
```
Hmmm -- this permission should actually be specified.  If it's not obvious from the code, then (a) check if the 23 branch version of this file contains better information and update accordingly, and if not, then (b) ping me to fill it out.  merci!

Reply:
This was discussed recently in https://github.com/aces/Loris/pull/6281#discussion_r415815975. Check that discussion; it will be helpful.